assets/js/modules/pagespeed-insights/components/dashboard/DashboardPageSpeed.js

```diff
@@ -150,6 +151,8 @@ export default function DashboardPageSpeed() {
 	} );
 	const inView = !! intersectionEntry?.intersectionRatio;
 
+	const isFetching = isFetchingMobile || isFetchingDesktop;
```

Comment:
I think we should consider only the currently selected strategy here because on the initial load it keeps showing the progress bar even though the data is loaded for the mobile view (which is selected by default).

Reply:
Thanks for clarifying this @eugene-manuilov 👍